features/destroy.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Trash and Untrash a message
+  As a user
+  In order to have a neat inbox
+  I should be able to delete messages
+
+Background:
```

Comment:
IMO You only really need a background step if you have more than one scenario that depends on the same steps before they're actually executed. In this case, you only have one scenario in this Feature so you can move the step `Given following user exists` to the body of the scenario